Packs/QueryAI/README.md

```diff
@@ -1,3 +1,12 @@
 ## Query.AI
-Query.AI is a decentralized data access and analysis technology that simplifies security investigations across disparate platforms, without data duplication.
+Query.AI provides the market’s only security investigations control plane for modern enterprises. Its patented browser-based platform delivers real-time access and centralized insights across on-premises, multi-cloud, and SaaS applications, without duplicating data from its native locations.
 
+##### What does this pack do?
+This integration helps in simplifying data querying and incident response by connecting to Query.AI’s unified interface.
+- Search for data in natural English language
+- Build playbooks to trigger different workflows in Query.AI
+- Get data from across disparate platforms through Query.AI’s decentralized API architecture
+
+Some useful resources :-
+- For more information, please visit us as at [Query.AI](https://query.ai)
```

Comment:
when we sasy 'us' it sounds like PANW XSOAR team. It will be better if we change it to just :

"For more information, please visit [Query.AI](https://query.ai)"

Reply:
Done.